test/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+find_package(Threads REQUIRED)
+
+# Enable ExternalProject CMake module
+include(ExternalProject)
+
+# Download and install GoogleTest
+ExternalProject_Add(
```

Comment:
probably it is good time to use just find_package(GTest)

Reply:
Thanks. However, let's keep it, just copy from calculator branch ;)